taipy/core/taipy.py

```diff
@@ -625,7 +625,7 @@ def get_jobs() -> List[Job]:
     return _JobManagerFactory._build_manager()._get_all()
 
 
-def delete_job(job: Job, force=False):
+def delete_job(job: Job, force: Optional[bool] = False):
```

Comment:
This method returns None, but somehow we do have the
```
return _JobManagerFactory._build_manager()._delete(job, force)
```

The `return` is redundant, which is confusing but very minor